crates/starknet-types-core/src/felt/mod.rs

```diff
@@ -219,15 +232,24 @@ impl Felt {
 
     /// Converts to little-endian bit representation.
     /// This is as performant as [to_bits_be](Felt::to_bits_be)
+    #[cfg(target_pointer_width = "64")]
+    pub fn to_bits_le(&self) -> BitArray<BitArrayStore> {
+        let limbs = self.0.representative().limbs;
+
+        BitArray::new(limbs)
+    }
+
+    /// Converts to little-endian bit representation.
+    /// This is as performant as [to_bits_be](Felt::to_bits_be)
+    #[cfg(all(feature = "alloc", not(target_pointer_width = "64")))]
     pub fn to_bits_le(&self) -> BitArray<BitArrayStore> {
         let limbs = self.0.representative().limbs;
 
-        #[cfg(not(target_pointer_width = "64"))]
         // Split limbs to adjust to BitArrayStore = [u32; 8]
         let limbs: [u32; 8] = limbs
             .into_iter()
             .flat_map(|n| [n as u32, (n >> 32) as u32])
-            .collect::<Vec<u32>>()
+            .collect::<alloc::vec::Vec<u32>>()
```

Comment:
import it at the top plz
```suggestion
            .collect::<Vec<u32>>()
```

Reply:
Look line 244. This is only available if the feature `alloc` is enabled.
And it's not used anywhere except in those functions.
It's better to have it here, where it is used, already behind the feature flag

Reply:
ah right